Packages/cc.starlessnight.unity-jsb/Source/Binding/Values_match.cs

```diff
@@ -127,7 +127,7 @@ public static bool js_match_type(JSContext ctx, JSValue jsValue, Type type)
                     return JSApi.JS_IsArray(ctx, jsValue) == 1;
                 }
 
-                return type.IsSubclassOf(typeof(ScriptValue));
+                return type == typeof(ScriptValue) || type.IsSubclassOf(typeof(ScriptValue));

```

Comment:
The two conditions can be written together as `typeof(ScriptValue).IsAssignableFrom(type)`